Mathlib/Data/Finset/Card.lean

```diff
@@ -760,12 +760,14 @@ theorem strongInductionOn_eq {p : Finset α → Sort*} (s : Finset α)
   rw [strongInduction]
 
 @[elab_as_elim]
-theorem case_strong_induction_on [DecidableEq α] {p : Finset α → Prop} (s : Finset α) (h₀ : p ∅)
+theorem caseStrongInductionOn [DecidableEq α] {p : Finset α → Prop} (s : Finset α) (h₀ : p ∅)
```

Comment:
This one shouldn't be renamed: the others have names following https://leanprover-community.github.io/contribute/naming.html, and this one did too, whereas the suggested change here breaks this convention.

Reply:
```suggestion
theorem case_strong_induction_on [DecidableEq α] {p : Finset α → Prop} (s : Finset α) (h₀ : p ∅)
```